utils/rpms/cart.spec

```diff
@@ -14,7 +14,7 @@ BuildRequires: scons >= 2.4
 BuildRequires: libfabric-devel
 BuildRequires: pmix-devel
 BuildRequires: openpa-devel
-BuildRequires: mercury-devel < 1.0.1-21
+BuildRequires: mercury-devel < 1.0.1-22
```

Comment:
Given that we are going to keep this one tightly specified, I wonder if we should just do = instead of <, since it really wouldn't work to use -20 either, would it?
```suggestion
BuildRequires: mercury-devel = 1.0.1-22
```